src/helpers/networkLocalStorage.ts

```diff
@@ -0,0 +1,22 @@
+import { LOCAL_STORAGE_NETWORK } from "constants/settings";
+
+interface NetworkLocalStorageSaveProps {
+  name: string;
+  horizonURL?: string;
+  networkPassphrase?: string;
+}
+
+export const networkLocalStorageSaveValue = (
+  values: NetworkLocalStorageSaveProps,
+) => {
+  try {
+    localStorage.setItem(LOCAL_STORAGE_NETWORK, JSON.stringify(values));

```

Comment:
this key/value pair doesn't match what currently exists in prod. I think we should keep it the same unless there's a really really good reason not to. I just don't want to introduce bugs because of an API change:
<img width="1601" alt="Screen Shot 2021-08-11 at 3 22 31 PM" src="https://user-images.githubusercontent.com/6789586/129090436-be25de4d-a99a-49de-91f9-623a660dc8f5.png">

Reply:
This change was intentional because we don't want to mutate the state directly, which the "old" way does. The update is to dispatch the same action as picking the network when the component mounts. Did you notice anything wrong there? I thought I tested it well, but I might have missed something.

Reply:
To clarify, we used to save the whole network store in local storage. The update makes the change dynamically by dispatching the action.

![image](https://user-images.githubusercontent.com/7346473/129092501-fd010076-cf69-4190-ae79-2bb83b083faf.png)